tests/git_stats_test.py

```diff
@@ -45,7 +50,7 @@ def test_pr_list(self):
         cmd_github = '!mergable {}'
         cmd_gitlab = '!mergable {}'
 
-        self.mock_repo.merge_requests.return_value = [mock_github_mr]
+        self.mock_repo.merge_requests = [mock_github_mr,]
```

Comment:
The trailing comma isnt needed here..
otherwise, this does look like an acceptable solution